environments/yoma-multitenant/aca-py-agent.default.env

```diff
@@ -55,17 +55,17 @@ ACAPY_MULTITENANT_JWT_SECRET=jwtSecret
 ACAPY_PUBLIC_INVITES=true
 
 # Optional Helper Configurations - See https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/config/argparse.py
-ACAPY_AUTO_ACCEPT_INVITES=false
-ACAPY_AUTO_ACCEPT_REQUESTS=false
+ACAPY_AUTO_ACCEPT_INVITES=true
+ACAPY_AUTO_ACCEPT_REQUESTS=true
```

Comment:
This doesn't seem in line with #154

Reply:
154 has:
```
auto-accept-invites: true
auto-accept-requests: true
```
in the ticket. I was wondering whether this si still what we wanted (had in mind that eg auto accept invites should be turned off). Just followed the ticket (#154)

Reply:
ah sorry my bad. It is in line with #154. I think we can leave it off for connections because we pass auto accept when creating / receiving the invite

Reply:
no worries. better to double check one too many times

Reply:
I'll turn it off then